docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,27 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+.. code-block:: text
```

Comment:
That fixes it. :)

Unfortunately, I just noticed the changes to `dpdk-guide.md`. There's a problem...